awx/sso/strategies/django_strategy.py

```diff
@@ -1,27 +0,0 @@
-# Copyright (c) 2017 Ansible, Inc.
-# All Rights Reserved.
-
-from social.strategies.django_strategy import DjangoStrategy
-
-
-class AWXDjangoStrategy(DjangoStrategy):
-    """A DjangoStrategy for python-social-auth containing
-       fixes and updates from social-app-django
-
-       TODO: Revert back to using the default DjangoStrategy after
-       we upgrade to social-core / social-app-django. We will also
-       want to ensure we update the SOCIAL_AUTH_STRATEGY setting.
-    """
-
-    def request_port(self):
-        """Port in use for this request
-           https://github.com/python-social-auth/social-app-django/blob/master/social_django/strategy.py#L76
-        """
-        try:  # django >= 1.9
-            return self.request.get_port()
-        except AttributeError:  # django < 1.9
-            host_parts = self.request.get_host().split(':')
-            try:
-                return host_parts[1]
-            except IndexError:
```

Comment:
In 3.2.2 branch we have a fix here that would catch some load-balancer corner cases:
```
+                if settings.USE_X_FORWARDED_PORT and 'HTTP_X_FORWARDED_PORT' in self.request.META:
+                    return self.request.META['HTTP_X_FORWARDED_PORT']
+                else:
+                    return self.request.META['SERVER_PORT']
```

Relates issue 7586, we may want to re-evaluate if this logic should be added.

Reply:
Underlying `get_port` implementation from Django 1.11 is doing this already.

https://docs.djangoproject.com/en/1.11/_modules/django/http/request/#HttpRequest.get_port